Directory.Packages.props

```diff
@@ -188,9 +188,9 @@
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.4" />
     <PackageVersion Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
     <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
-    <PackageVersion Include="Azure.Core" Version="1.42.0" />
+    <PackageVersion Include="Azure.Core" Version="1.43.0" />
```

Comment:
This is necessary because the new Azure.Provisioning depends on 1.43 and so CPM freaks out if you have 1.42 pinned globally.